soc/arm/nordic_nrf/nrf53/Kconfig.soc

```diff
@@ -47,6 +49,25 @@ config SOC_NRF53_ANOMALY_160_WORKAROUND
 	depends on SYS_CLOCK_EXISTS
 	select ARM_ON_ENTER_CPU_IDLE_HOOK
 
+config SOC_NRF53_RTC_PRETICK
+	bool
```

Comment:
Imo it should have prompt. Otherwise you cannot do
`SOC_NRF53_RTC_PRETICK=n` in project config. Many may want to disable it given that in room temperature issue is so extremely rare.

Reply:
done